index.js

```diff
@@ -0,0 +1,80 @@
+const stream = require('stream');
+const streamMap = require('through2-map');
+
+module.exports = (knex, config) => {
+  const columnName = config.columnName || 'value';
+
+  async function open () {
+    return knex.schema.createTableIfNotExists(config.tableName, function (table) {
+      table.increments();
+      table.jsonb(columnName);
+      table.timestamps(true, true);
+    }).then(() => {
+      if (config._purgeLog) return knex(config.tableName).delete();
+    });
+  }
+
+  async function close () {
+    return Promise.resolve();
+  }
+
+  async function append (payload) {
+    const data = {};
+    data[columnName] = JSON.stringify(payload);
+    return knex(config.tableName)
+      .insert(data)
+      .returning('id')
+      .then((inserted) => {
+        return knex(config.tableName).first('id', 'created_at')
+            // NOTE (tim): sqlite cannot do returning of multiple cols, so we
+            // have to requery to get the created_at.
+            .where({ id: inserted[0] })
+            .then((result) => {
+              return { id: result.id, timestamp: result.created_at };
+            });
+      });
+  }
+
+  async function get (offset) {
+    return knex(config.tableName).first('id', columnName)
+      .where({ id: offset.id })
```

Comment:
might want a guard statement in case `offset.id` is not present and throw error. But not the end of the world.